docs/pages/build/building-on-ci.mdx

```diff
@@ -8,17 +8,21 @@ import { Terminal } from '~/ui/components/Snippet';
 
 This document outlines how to trigger builds on EAS for your app from a CI environment such as GitHub Actions, Travis CI, and more.
 
-Before building with EAS on CI, we need to install and configure `eas-cli`. Then, we can trigger new builds with the `eas build` command.
-
 ## Prerequisites
 
 ### Run a successful build from your local machine
 
-To trigger EAS builds from a CI environment, we first need to configure our app for EAS Build and successfully run a build from our local machine for each platform that we'd like to support on CI.
+To trigger EAS builds from a CI environment, your app needs to be setup to use EAS Build in non-interactive mode. The  way to do this is by taking care of the EAS Build initialization steps and running a successful build from your local terminal for each platform you would like to support on CI. This way, the `eas build` command can prompt interactively for any additional configuration it needs, and then that configuration will be available for future non-interactive runs on CI.
+
+Running a build locally will accomplish the following critical configuration steps:
+- Initialize the project on EAS, generating a `projectId`.
+- Add an **eas.json** file defining your build profiles.
+- Ensure app config fields critical for native builds are populated, such as Android `packageName` and iOS `bundleIdentifier`.
```

Comment:
```suggestion
- Populates critical app config properties for native builds, such as `android.packageName` and `ios.bundleIdentifier`.
```
- On the [app config reference page](https://docs.expo.dev/versions/latest/config/app/), we use "properties instead of "fields". Sticking to the same terminology helps the reader to follow the connection.
- Should we use the app config field names directly and include the platform properties as part of the syntax?
- IIRC, this initialization step also prompts to add `android.package` and/or `bundleIdentifier` if they are not set already. So we can improve this sentence's readability?